neo/Consensus/RecoveryMessage.ChangeViewPayloadCompact.cs

```diff
@@ -10,20 +10,20 @@ public class ChangeViewPayloadCompact : ISerializable
         {
             public ushort ValidatorIndex;
             public byte OriginalViewNumber;
-            public uint Timestamp;
+            public long Timestamp;
```

Comment:
Why `long` and not `ulong`?

Reply:
I think `long` opens doors to some unexpected attacks (negative timings), we should adopt `ulong` I think.